ui/src/containers/appView/AppViewContainer.js

```diff
@@ -36,11 +42,30 @@ class AppViewContainer extends React.Component {
     // to update the App versions
   }
 
+  isAppVersionsDirty () {
+    const currentVersions = this.props.app.deployedVersions;
```

Comment:
```suggestion
{ currentVersions, savedVersions } = this.props;
```

Reply:
How does this work @craicoverflow? Would this not expect the variables to exist as `this.props.currentVersions` and `this.props.savedVersions`?